lib/surface/components/form/checkbox.ex

```diff
@@ -20,13 +20,30 @@ defmodule Surface.Components.Form.Checkbox do
   import Phoenix.HTML.Form, only: [checkbox: 3]
   import Surface.Components.Form.Utils
 
+  @doc "The value to be sent when the checkbox is checked. Defaults to \"true\""
+  prop checked_value, :any, default: true
+
+  @doc "Controls if this function will generate a hidden input to submit the unchecked value or not, defaults to \"true\"."
+  prop hidden_input, :boolean, default: true
+
+  @doc "The value to be sent when the checkbox is unchecked, defaults to \"false\"."
+  prop unchecked_value, :any, default: false
+
   def render(assigns) do
-    props = get_non_nil_props(assigns, [:checked, class: get_config(:default_class)])
+    props =
+      get_non_nil_props(assigns, [
+        :checked_value,
+        :hidden_input,
+        :unchecked_value,
+        :checked,
```

Comment:
I think we can remove `:checked` from that list since we'll be using `:value` instead.

Reply:
Thanks for the review! This has been updated in the forked branch.